translations/da/get5.phrases.txt

```diff
@@ -0,0 +1,383 @@
+"Phrases"
+{
+    "ReadyToVetoInfoMessage"
+    {
+        "da"            "Skriv {GREEN}!ready{NORMAL} når dit hold er klar til at veto."
+    }
+    "WaitingForCastersReadyInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "da"            "Venter på {1} til at skrive {GREEN}!ready {NORMAL}for at begynde."
```

Comment:
Hey. Der er ikke rigtigt noget, der hedder "venter på x til at...".
`Venter på, at x skriver !ready for at begynde.` ville nok være et bedre bud.
Med mindre jeg forstår sætningen her forkert.

Reply:
Det er nok mere korrekt ja, tror også selv jeg ikke helt forstod linjen. Men dette giver bedre mening!